charmcraft.yaml

```
@@ -26,6 +26,12 @@ bases:
       channel: "22.04"
       architectures: ["arm64"]
 
+charm-libs:
+  - lib: "tls_certificates_interface.tls_certificates"
+    version: 3
+  - lib: "observability-libs.cert_handler"
+    version: 1
+
 parts:
   charm:
     plugin: charm
```

I get this error when packing on charmcraft 3.2.1

```bash
charmcraft internal error: AttributeError("'int' object has no attribute 'partition'")
```
removing charm-libs and changing parts to be like below seems to fix the issue

```suggestion
parts:
  charm:
    charm-binary-python-packages:
      - "pydantic>=2"
      - "cryptography"
      - "jsonschema"
```

---

Seems like charmcraft change how it lints the charm-libs version field between 3.1.2 and 3.2.1.  I believe it is now fixed, but let me know if it still doesn't work for you.  

background: [this charmcraft issue](https://github.com/canonical/charmcraft/issues/1954)